# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openoffice-org/patches/jvmfwk-gij-paths.diff
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- jvmfwk/plugins/sunmajor/pluginlib/gnujre.cxx	2008-04-12 08:54:21.000000000 +0000
+++ jvmfwk/plugins/sunmajor/pluginlib/gnujre.cxx.new	2008-06-11 10:25:37.000000000 +0000
@@ -70,6 +70,22 @@
         "/libjvm.so",
         "/lib/" GCJ_JFW_PLUGIN_ARCH "/client/libjvm.so",
         "/gcj-4.1.1/libjvm.so",
+        "/gcj-4.2/libjvm.so",
+        "/gcj-4.1/libjvm.so",
+        "/gcj-4.2.0/libjvm.so",
+        "/gcj-4.2.1/libjvm.so",
+        "/gcj-4.2.2/libjvm.so",
+        "/gcj-4.2.2-81/libjvm.so",
+        "/gcj-4.2.3-81/libjvm.so",
+        "/gcj-4.3.0-9/libjvm.so",
+        "/gcj-4.3.1-9/libjvm.so",
+        "/gcj-4.1.2/libjvm.so",
+        "/libgcj.so.80",
+        "/libgcj.so.81",
+        "/libgcj.so.8",
+        "/libgcj.so.71",
+        "/libgcj.so.70",
+        "/libgcj.so.9",
         "/libgcj.so.7",
         "/libgcj.so.6"
     };
